src/main/webapp/user-info.html

```diff
@@ -5,10 +5,11 @@
     <link href="https://cdn.jsdelivr.net/npm/vuetify/dist/vuetify.min.css" rel="stylesheet">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
     <meta charset="UTF-8">
+    <script src="/js/navigation-loader.js"></script>
     <script src="/js/user-info-loader.js"></script>
     <title>Skill Search</title>
 </head>
-<body>
+<body onload="addLoginOrLogoutLinkToNavigation()">
```

Comment:
After I had a short discussion with Alison, we thought that user-info couldn't be shown on the navigation bar when the user was log out.
I think the function, addLoginOrLogoutLinkToNavigation(), should be edited. It should removed user-info link when the user is log out.

Reply:
OK, I'll change that.

Reply:
@Sinyu104 I think we should remove the user-info from nav bar completely, and use an edit button on personal page to link to user-info.html directly.

Reply:
okay!